lib/msf/core/db_manager/import.rb

```diff
@@ -484,8 +484,10 @@ def import_filetype_detect(data)
   def msf_import_timestamps(opts,obj)
     obj.created_at = opts["created_at"] if opts["created_at"]
     obj.created_at = opts[:created_at] if opts[:created_at]
+    obj.created_at = opts[:task].created_at if opts[:task]&.created_at
```

Comment:
Does this fix belong in Pro? I don't think Metasploit's core knows anything about 'tasks'